napari/_app_model/actions/_view_actions.py

```diff
@@ -0,0 +1,58 @@
+"""Actions related to the view that require Qt.
+
+View actions that do not require Qt should go in
+napari/_app_model/actions/_view_actions.py.
+"""
+
+from typing import List
+
+from app_model.types import Action
+
+from ...settings import get_settings
+from ..constants import CommandId, MenuId
+from ._toggle_action import ViewerToggleAction
+
+VIEW_ACTIONS: List[Action] = []
+
+for cmd, viewer_attr, sub_attr in (
+    (CommandId.TOGGLE_VIEWER_AXES, 'axes', 'visible'),
+    (CommandId.TOGGLE_VIEWER_AXES_COLORED, 'axes', 'colored'),
+    (CommandId.TOGGLE_VIEWER_AXES_LABELS, 'axes', 'labels'),
+    (CommandId.TOGGLE_VIEWER_AXES_DASHED, 'axes', 'dashed'),
+    (CommandId.TOGGLE_VIEWER_AXES_ARROWS, 'axes', 'arrows'),
+    (CommandId.TOGGLE_VIEWER_SCALE_BAR, 'scale_bar', 'visible'),
+    (CommandId.TOGGLE_VIEWER_SCALE_BAR_COLORED, 'scale_bar', 'colored'),
+    (CommandId.TOGGLE_VIEWER_SCALE_BAR_TICKS, 'scale_bar', 'ticks'),
+):
+    menu = MenuId.VIEW_AXES if viewer_attr == 'axes' else MenuId.VIEW_SCALEBAR
+    VIEW_ACTIONS.append(
+        ViewerToggleAction(
+            id=cmd,
+            title=cmd.title,
+            viewer_attribute=viewer_attr,
+            sub_attribute=sub_attr,
+            menus=[{'id': menu}],
+        )
+    )
+
+
+def _tooltip_visibility_toggle():
+    settings = get_settings().appearance
+    settings.layer_tooltip_visibility = not settings.layer_tooltip_visibility
+
+
+VIEW_ACTIONS.extend(
+    [
+        # TODO: this could be made into a toggle setting Action subclass
+        # using a similar pattern to the above ViewerToggleAction classes
+        Action(
+            id=CommandId.TOGGLE_LAYER_TOOLTIPS,
+            title=CommandId.TOGGLE_LAYER_TOOLTIPS.title,
+            menus=[
+                {'id': MenuId.MENUBAR_VIEW, 'group': '1_render', 'order': 10}
+            ],
+            callback=_tooltip_visibility_toggle,
+            toggled='settings_appearance_layer_tooltip_visibility',  # TODO
```

Comment:
I tried to understand what this string is doing, but I don't really get it. The only other assignment to `toggled` is a `ToggleRule`, which I understand as being the _"Condition under which the command should appear checked/toggled in any GUI representation"_ (as per the `CommandRule` documentation), but I don't really understand how this string is achieving that?

Reply:
> I tried to understand what this string is doing, but I don't really get it.

lol ... probably not a d@µn thing :joy:   i guess I missed this one.   was really hoping not to have to deal with things that have to synchronize with settings here.

Reply:
@tlambert03 just circling back on this one because I realized that even though the `ToggleRule` is just a string, the checked/unchecked behaviour is still working? Do you know where in the code that's being handled now? For the activity dialog for example, we had to add a real `ToggleRule` to see the checkmark - but for these tooltips it's just working...?

Reply:
Ok so I think @lucyleeow and I have worked it out.

If you pass in `False` to the `toggled` attribute of the action, the menu item starts off unchecked, and will be flipped every time the `Action` is triggered.

If you pass in `True` to the `toggled` attribute, the menu item starts off checked, and will be flipped every time the `Action` is triggered.

If you pass in a `callable` to the `toggled` attribute, the return value of the callable will determine whether or not the menu item is checked.

Reply:
Note this (toggle activity doc) has been updated to a callable.

There are 2 other view items: 'Toggle Play' and 'Toggle Full Screen' where the check mark does not appear - this is the case in main as well so it was decided to be outside the scope of this PR. It can be amended later for these 2 items.